unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExploreHUD/ExploreHUDView.cs

```diff
@@ -10,6 +10,8 @@ internal class ExploreHUDView : MonoBehaviour
     [SerializeField] internal Button_OnPointerDown togglePopupButton;
     [SerializeField] internal Color[] friendColors = null;
 
+    public event System.Action OnClose;
```

Comment:
In the same way we are doing in other views like `WorldChatWindowHUDView`, `PrivateChatWindowHUDView`, `FriendsHUDView`, etc. the responsibility of notify about the window closing is of the own view and not the controller (anyway we can discuss it with the team in order to do a possible refactor in all classes). Anyway, even moving this event to the controller, we would continue needing to be public because it the class is used in other test classes like `TaskbarHUDShould`, so I cannot convert it to internal :'(

Reply:
@sandrade-dcl By using the `[assembly: InternalsVisibleTo("<assembly-name>")]` we can make the internals visible to the testing classes, so maybe you can use this to keep the internal class?. Just look for any `AssemblyInfo.cs` file. This is a pattern we currently use quite broadly!.

Reply:
Ok, I will take a look to it, thanks guys.

Reply:
Done! Thanks to the AssemblyInfo, I have been able to keep the class as internal, just as we wanted.